packages/edge-login-ui-rn/src/components/screens/newAccount/NewAccountReviewScreen.js

```diff
@@ -34,10 +34,22 @@ class NewAccountReviewScreenComponent extends React.Component<Props> {
               </Text>
             </View>
             <View style={styles.warningBoxContainer}>
-              <WarningBox
-                style={styles.warningBox}
-                message={s.strings.warning_message}
-              />
+              <View style={styles.warningBox.container}>
+                <View style={styles.warningBox.bottom}>
+                  <Text style={styles.warningBox.text}>
+                    {s.strings.warning_message}
+                  </Text>
+                </View>
```

Comment:
Can't you just merged the styles from the View into the Text and have only the Text component?

Reply:
Tried it, but it weirdly broke the layout.